From 9b1bc862f76b3fe41ea3551234e4ef5b94c1a03e Mon Sep 17 00:00:00 2001
From: Feng Zhang <feng@radxa.com>
Date: Fri, 30 Jan 2026 03:37:33 +0000
Subject: [PATCH 5/9] defconfig: enable FDT_FIXUP_PARTITIONS for Cubie A7S

Enable CONFIG_FDT_FIXUP_PARTITIONS in radxa-cubie-a7s_defconfig to allow
runtime partition table fixup in the device tree (e.g., for U-Boot or
kernel-based flash layout adjustments).
---
 configs/radxa-cubie-a7s_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/configs/radxa-cubie-a7s_defconfig b/src/configs/radxa-cubie-a7s_defconfig
index 0ae73a5dbd9..1cd767f290c 100644
--- a/src/configs/radxa-cubie-a7s_defconfig
+++ b/src/configs/radxa-cubie-a7s_defconfig
@@ -10,6 +10,7 @@ CONFIG_MMC_SUNXI_SLOT_EXTRA=2
 # CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
 CONFIG_SPL_LOAD_FIT_ADDRESS=0x48200000
 CONFIG_BOOTDELAY=3
+CONFIG_FDT_FIXUP_PARTITIONS=y
 CONFIG_SPL_RAM_DEVICE=y
 CONFIG_CMD_DFU=y
 CONFIG_CMD_I2C=y
-- 
2.49.0

